backend/vodabackend/vodaMainApp/urls.py

```diff
@@ -0,0 +1,16 @@
+# temp filler from django tutorial
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.map, name='map'),
+    path('<int:supply_id>/summary', views.summary, name='summary'),
```

Comment:
This isn't how you do query parameters in django. https://stackoverflow.com/questions/150505/capturing-url-parameters-in-request-get